torch_glow/tests/nodes/quantized_linear_test.py

```diff
@@ -5,39 +5,14 @@
 from tests.utils import jitVsGlow
 
 
-def test_quantized_linear():
```

Comment:
For my own curiosity, why remove this test?

Reply:
This was basically a placeholder test for linear with some constant inputs that gave accurate enough results. With the added FC node, the accuracy is good enough that we can switch to testing with a bunch of random inputs which is a better test.